src/record.py

```diff
@@ -0,0 +1,44 @@
+"""Record class."""
+from parse import get_parser, get_card
+from extract import read_to_dict
+from standardize import standardizer, strip_payment
+from common import flip_spending_sign
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+class CardRecord(object):
+    def __init__(self, filename):
+        self.filename = filename
+        self.card = get_card(filename)
+        self.parser = get_parser(filename)
+
+    def read_expenses(self):
+        records = read_to_dict(self.filename)
```

Comment:
It looks like this starts from reading the csv. What do you think about pointing it at the final sqlite database instead? That way it can automatically pick up multiple cards if there are multiple card types. Also this class won't need to think about data cleaning / parsing since all that would already be done before this layer.

Something like `df = pd.from_sql("expenses.db")` ?

Should we want to look at just 1 card, instead of all, we can filter on the source column to just `amex` for example.

Reply:
So I would like a pure pythonic way to read expenses. I think a bash script to trigger multiple python scripts to populate a db and then read from that db is somewhat circular. Especially right now that the database is static, and has to be manuly generated.

This particular class is meant as a record for a single card. I'd probably want a subclas or a different class which includes all expenditure.